src/components/MegaButton/MegaButton.stories.js

```diff
@@ -78,7 +78,7 @@ const GroupTemplate = (args, { argTypes }) => ({
   setup: () => ({ args }),
   data: () => ({ megaButtonModel }),
   template: `
-    <RadioGroup legend="Cat Type">
```

Comment:
I don't think we should remove the RadioGroup from here, as we do want the megas to work in the group?